vscode/webviews/Chat.tsx

```diff
@@ -209,43 +211,54 @@ const TextArea: React.FunctionComponent<ChatUITextAreaProps> = ({
         }
     }, [])
 
-    // <VSCodeTextArea autofocus> does not work, so implement autofocus ourselves.
-    useEffect(() => {
-        if (autoFocus) {
-            inputRef.current?.focus()
-        }
-    }, [autoFocus])
+    const onTextAreaKeyDown = useCallback(
+        (event: React.KeyboardEvent<HTMLElement>): void => {
+            if (onKeyDown) {
+                onKeyDown(event, inputRef.current?.selectionStart ?? null)
+            }
```

Comment:
Typescript has a shorthand for this. I'm not sure about the semantics of argument evaluation but that should not matter here.

```suggestion
            onKeyDown?.(event, inputRef.current?.selectionStart ?? null)
```